.env

```diff
@@ -3,3 +3,4 @@ FLASK_DEBUG=true
 SECRET_KEY=secret_key
 DEVEL=True
 OPENID_LAUNCHPAD_TEAM=canonical-content-people
+DATABASE_URL=postgresql://postgres:pw@localhost/db
```

Comment:
are we checking in the password here? If you have access to the host node, you could set environment variables there instead of checking in the `.env` file, or maybe IS has some external secrets manager capability?

Reply:
We check in the passwords in our `.env.local` file. It's not checked into git. `.env.local` file overrides env vars in `.env` file.

Reply:
Sure, so the purpose of `.env` checked in here is to serve as a reminder that `.env.local` is needed? I see this is documented in `README.md`, all good then :+1: